consul-l7-observability-kubernetes/step4.md

```diff
@@ -0,0 +1,47 @@
+
+Install Grafana with the following command.
+
+`kubectl apply -f app`{{execute interrupt T1}}
+
+You should receive output similar to the following:
+
+```plaintext
+service/frontend-service created
+...TRUNCATED
+deployment.apps/public-api created
+```
+
+### Verify the installation
+
+Verify the installation using the following command:
+
+`watch kubectl get pods`{{execute T1}}
+
+Once all pods have a status of `Running` the installation is complete.
+
+```plaintext
+NAME                                                         READY   STATUS    RESTARTS   AGE
+...
+frontend                                                     3/3     Running   0          120m
+...
+postgres                                                     3/3     Running   0          120m
+products-api                                                 3/3     Running   0          120m
+...
+public-api                                                   3/3     Running   0          120m
+```
+
+Next, in a different terminal, forward port `80` from the `frontend` pod hosting the UI to the development
```

Comment:
Since this is a beginner lab, can we add quick instructions on how to get another terminal window?

Reply:
I'm actually doing it for them.  I'll update this language to make that super obvious. Good catch.